src/com/teamcautionrobotics/robot2018/Climb.java

```diff
@@ -0,0 +1,39 @@
+package com.teamcautionrobotics.robot2018;
+
+import edu.wpi.first.wpilibj.VictorSP;
+
+public class Climb {
+
+    private VictorSP climbMotor;
+
+    public Climb(int motorPort) {
+
+        climbMotor = new VictorSP(motorPort);
+
+    }
+
+    public void ascend() {
```

Comment:
Depending on how the rest of the code ends up, we may want to from separate `ascend` and `descend` functions to `void move(Direction direction)`, like in our [2016 robot code](https://github.com/TeamCautionRobotics/2016-FRC-robot/blob/master/src/org/usfirst/frc/team1492/robot/Robot.java#L356).
I do not recommend this change now, though this may be a useful change later.